### PR TITLE
Fixing time series

### PR DESCRIPTION
**Pull request by sourabhdalvi**

### CODE REVIEW

File: src/system_library.jl [hunk 1]
```diff
@@ -133,22 +133,6 @@ function build_c_sys5_re(; kwargs...)
         end
     end
 
-    if get(kwargs, :add_single_time_series, false)
```
Comment (jd-lara) [inline src/system_library.jl:136]: Why are you removing this?

**Comment by jd-lara** [comment]
@sourabhdalvi does this duplicates #23 also?

File: src/system_library.jl (hunk 1 above)
Comment (sourabhdalvi) [reply src/system_library.jl:136]: Ignore this for now, there are few other changes in here too, once your PR is merged I'll close this and open a new one with the relevant changes

Comment (jd-lara) [reply src/system_library.jl:136]: For the tests to pass we also need a new IS release

**Pull request merged**